orcoursetrion/lib/github.py

```diff
@@ -134,7 +134,7 @@ def add_team_repo(self, org, repo, team):
             raise GitHubUnknownError(
                 'No teams found in {0} organization'.format(org)
             )
-        found_team = [x for x in teams if x['name'] == team]
+        found_team = [x for x in teams if x['name'].strip() == team.strip()]
```

Comment:
Thanks for putting this in here, hopefully I won't have to add teams again; but it's nice to be able to sort them with leading spaces.